backend/api/Controllers/DeckController.cs

```diff
@@ -220,5 +222,68 @@ public async Task<ActionResult<DeckResponse>> DeleteDeck([FromRoute] string id)
                 return NotFound($"Deck with id {id} not found");
             return Ok(new DeckResponse(deck));
         }
+
+        /// <summary>
+        /// Gets map metadata for localization poses belonging to deck with specified id
+        /// </summary>
+        [HttpGet]
+        [Authorize(Roles = Role.Any)]
```

Comment:
Should it not be a user and then admin can do this automatically or does it not works that way ?

Reply:
Seems that we have very mixed use of 'any' and 'user', and that 'any' is most used. Suggest that we make a card on being more consistent and solve that separately